packages/sdk/src/clinical/prescription.ts

```diff
@@ -84,29 +84,29 @@ export class PrescriptionQueryManager {
     }
     const [fName, fValue] = Object.entries(fragment)[0];
     const GET_PRESCRIPTIONS = gql`
-          ${fValue}
-          query prescriptions(
-            $patientId: ID
-            $patientName: String
-            $prescriberId: ID
-            $state: PrescriptionState
-            $after: ID
-            $first: Int
-        ) {
-            prescriptions(
-                filter: {
-                    patientId: $patientId
-                    patientName: $patientName
-                    prescriberId: $prescriberId
-                    state: $state
-                }
-                after: $after
-                first: $first
-            ) {
-                ...${fName}
-            }
+      ${fValue}
```

Comment:
hm I'd really not like to use this pattern any more if possible, unless there is a customer use case.